docs/setup.md

```diff
@@ -231,3 +236,84 @@ host directory.
 
 [netconf]: https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md#network-configuration
 [BIRD]: https://bird.network.cz/
+
+## Standalone egress
+
+Coil can be run as standalone egress NAT controller, using CNI chaining with another CNI providing base connectivity. This chapter will guide you on how to achieve this.
+
+### Configuration
+
+To deploy Coil with only egress feature enabled the following changes are required in the configuration files:
+
+1. Comment all IPAM related pieces in the following `kustomization.yaml` files:
+    - `v2/config/crd/kustomization.yaml`
+    - `v2/config/default/kustomization.yaml`
+    - `v2/config/pod/kustomization.yaml`
+    - `v2/config/rbac/kustomization.yaml`
+
+1. Comment unnecessary resources in `config/crd/patches/remove_status.yaml`.
+1. Add following arguments to the `coild` contianer executable in `config/pod/coild.yaml`
+    ```yaml
+    containers:
+      - name: coild
+        image: coil:dev
+        command: ["coild"]
+        args:
+          - --zap-stacktrace-level=panic
+          - --enable-ipam=false
+          - --enable-egress=true
+          - --pod-table-id=0 # 255 if IPv6 is being used
+          - --protocol-id=2
+    ```
+1. Set CNI config filename using environment variable for init contianer `coil-installer` in `config/pod/coild.yaml`:
+    ```yaml
+    env:
+    - name: CNI_CONF_NAME
+      value: "01-coil.conflist"
+    ```
+1. Set coil capabilites in `v2/netconf.json` to:
+    ```json
+    {
+      "type": "coil",
+      "socket": "/run/coild.sock",
+      "capabilities": {
+        "ipam": false,
+        "egress": true
+      }
+    },
+    ```
+1. Add configuration of your chosen CNI to `v2/netconf.json` before `coil` related configuration.
+1. Deploy `coil` to existing cluster as described in [Compile and apply the manifest](#compile-and-apply-the-manifest).
+
+### Testing standalone egress
+
+1. Configure Coil to run in standalone egress mode as described in [configuration](#configuration)
+1. Set proper CNI netconf to use in `e2e/kustomization.yaml`.
```

Comment:
In the future, we want to run this test in CI.
Therefore, we want to reduce manual procedures.

How about preparing `kustomization.yaml` for each CNI setting and switching the environment to use with an env value?

Reply:
I've created separate kustomization YAMLs and added proper targets to the test Makefile. Instructions in `docs/setup.md` were updated accordingly.